lib/src/screens/receive/widgets/address_cell.dart

```diff
@@ -71,7 +71,7 @@ class AddressCell extends StatelessWidget {
           ),
         ));
     return Semantics(
-      label: 'Slidable',
+      label: S.of(context).widgets_address,
```

Comment:
?

Reply:
It was supposed to indicate it is slidable but it would be discarded after the PR to replace swipes with edit buttons